### PR TITLE
Refactor: Use ethers namehash util

### PR DESCRIPTION
**Pull request by 0xGabi**
Use ethers namehash util and remove `eth-ens-namehash` dependency.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,6 @@
   "dependencies": {
     "chalk": "^3.0.0",
     "chokidar": "^3.3.1",
-    "eth-ens-namehash": "^2.0.8",
```
Comment (sohkai) [inline package.json:32]: Haha, I wonder if we should regenerate the package-lock (or even just move entirely to yarn...).

I would suspect more dependencies to be dropped from the lock file if we've removed this dependency.

File: src/utils/generateArtifacts.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ function decorateEnvrionmentsWithAppId(
   for (const [key, value] of Object.entries(environments)) {
     decoratedEnvrionment[key] = {
```
Comment (sohkai) [inline src/utils/generateArtifacts.ts:38]: Nitpick: `Environment` is misspelled.

File: src/utils/generateArtifacts.ts [hunk 3]
```diff
@@ -37,7 +37,7 @@ function decorateEnvrionmentsWithAppId(
...
       ...value,
-      appId: namehash.hash(value.appName)
+      appId: value.appName && namehash(value.appName)
```
Comment (sohkai) [inline src/utils/generateArtifacts.ts:40]: Every app should have an appId... so I'm wondering if we shouldn't catch and error in the case of an `appName` not being present?

File: package.json (hunk 1 above)
Comment (0xGabi) [reply package.json:32]: Move to yarn on #112

File: src/utils/generateArtifacts.ts (hunk 2 above)
Comment (0xGabi) [reply src/utils/generateArtifacts.ts:38]: We remove this piece of logic on the plugin. But I noted this to fix it on the CLI.

File: src/utils/generateArtifacts.ts (hunk 3 above)
Comment (dapplion) [reply src/utils/generateArtifacts.ts:40]: Fixed by https://github.com/aragon/buidler-aragon/pull/115

**Pull request merged**